Src/Campus.Master.API/UI/src/app/core/left-sidebar/left-sidebar.component.sass

```diff
@@ -1,4 +1,4 @@
-@import "src/styles/styles"
+@import "../../../styles/styles"
```

Comment:
Use path alias in style preprocessing. Here is an [example](https://stackoverflow.com/questions/42865697/scss-import-relative-to-root)

Reply:
Added !